src/jit/codegenarm64.cpp

```diff
@@ -3875,7 +3875,14 @@ void CodeGen::genSIMDIntrinsic(GenTreeSIMD* simdNode)
         simdNode->gtSIMDBaseType != TYP_SHORT && simdNode->gtSIMDBaseType != TYP_BYTE &&
         simdNode->gtSIMDBaseType != TYP_UINT && simdNode->gtSIMDBaseType != TYP_ULONG)
     {
```

Comment:
What do we have as a `simdNode->gtSIMDBaseType` in this case?

Reply:
It is `TYP_UNDEF`. I'm looking into this further, but thought it was worthwhile to merge this fix, since it is safe, to make r2r testing clean.